CHANGES.rst

```diff
@@ -6,6 +6,9 @@ Verison 0.5.0 [unreleased]
 
 WIP
 
+- `OPENWISP_NETWORK_TOPOLOGY_API_AUTH_REQUIRED <https://github.com/openwisp/openwisp-network-topology#openwisp-network-topology-api-auth-required>`_:
+  [API] Make API endpoints optionally require authentication and permissions.
```

Comment:
please try to adopt this change log style:

- https://github.com/openwisp/openwisp-users/blob/master/CHANGES.rst#version-030-2020-08-14
- https://github.com/openwisp/openwisp-utils/blob/master/CHANGES.rst#version-060-2020-08-14